test/suite/test_rollback_to_stable06.py

```diff
@@ -129,20 +129,13 @@ def test_rollback_to_stable(self):
         else:
             self.assertGreaterEqual(upd_aborted + hs_removed + keys_removed, nrows * 4)
 
-        if not self.in_memory and self.key_format == 'i':
-            # FIXME-WT-10017: WT-9846 is a temporary fix only for row store and a 
-            # more complete fix including column store will be made in WT-10017.
-            # Once delivered the key_format == 'i' check is no longer needed.
-            # 
-            # Reinsert the same updates with the same timestamps and flush to disk.
-            # If the updates have not been correctly removed by RTS WiredTiger will 
-            # see the key already exists in the history store and abort. 
-            self.large_updates(uri, value_a, ds, nrows, self.prepare, 20)
-            self.large_updates(uri, value_b, ds, nrows, self.prepare, 30)
-            self.large_updates(uri, value_c, ds, nrows, self.prepare, 40)
-            self.large_updates(uri, value_d, ds, nrows, self.prepare, 50)
-
-            self.session.checkpoint()
+        # Reinsert the same updates with the same timestamps and flush to disk.
+        # If the updates have not been correctly removed by RTS WiredTiger will 
+        # see the key already exists in the history store and abort. 
+        self.large_updates(uri, value_a, ds, nrows, self.prepare, 20)
+        self.large_updates(uri, value_b, ds, nrows, self.prepare, 30)
+        self.large_updates(uri, value_c, ds, nrows, self.prepare, 40)
+        self.large_updates(uri, value_d, ds, nrows, self.prepare, 50)
```

Comment:
Can you enhance the test to add `checkpoint` and `eviction` scenarios also, so that it may expose any other code flows?

Reply:
I don't understand. So you want me to add a checkpoint before or after?

Reply:
There is one checkpoint before, and I want you to add another one also at the end. Also the `debug=(release_evict)` option eviction after the checkpoint with options, so sometimes the data exist and in other cases, it doesn't.